modules/custom-post-types/portfolios.php

```diff
@@ -65,6 +65,14 @@ function __construct() {
 		add_shortcode( 'portfolio',                                                    array( $this, 'portfolio_shortcode' ) );
 		add_shortcode( 'jetpack_portfolio',                                            array( $this, 'portfolio_shortcode' ) );
 
+		if ( defined( 'IS_WPCOM' ) && IS_WPCOM ) {
```

Comment:
I'm not sure which is faster in this case: check for `IS_WPCOM`, or let WordPress check the filter.